code/modules/research/designs/weapons.dm

```diff
@@ -257,3 +257,138 @@
 	materials = list(MAT_IRON = 12000)
 	category = "Weapons"
 	build_path = /obj/item/weapon/storage/pneumatic
+
+//Ammolathe designs. They lack research values so they can't be researched.
+//Weapons
+
+/datum/design/glock
+	name = "NT Glock"
+	desc = "A NT Glock. It uses .380AUTO rounds."
+	id = "glock"
+	build_type = AMMOLATHE
+	materials = list(MAT_IRON = 10000, MAT_GLASS = 10000)
+	build_path = /obj/item/weapon/gun/projectile/sec/lockbox
+	locked = TRUE
+	req_lock_access = list(access_armory, access_weapons)
+
+/datum/design/vector
+	name = "Vector"
+	desc = "A gun."
```

Comment:
🤔

Reply:
I'm retarded and forgot to save/push changes one time it seems.

Reply:
Fixed.